### DISCUSSION TITLE
Pane jumping

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/terminal#12559] [Q&A] [answered]
I'm not sure if this is already implemented but jumping panes in a single window would be great and would make the workflow faster.

**Comment by zadjii-msft** [comment, marked as answer]
You mean like [Switching between panes](https://docs.microsoft.com/en-us/windows/terminal/panes#switching-between-panes)? Or maybe [swapping panes](https://docs.microsoft.com/en-us/windows/terminal/panes#swapping-panes)?

**Reply by ghost** [reply]
There it is. Yes. Thank you. (Should've read the entire docs.)
